lib/ansible/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -99,6 +99,16 @@
     record_type: 'AAAA'
     record_ip: '::1'
 
+# Ensure a PTR record is present
+- ipa_dnsrecord:
+    ipa_host: spider.example.com
+    ipa_pass: Passw0rd!
+    state: present
+    zone_name: 2.168.192.in-addr.arpa
+    record_name: 5
+    record_type: 'PTR'
+    record_ip: 'internal.ipa.example.com'
```

Comment:
I didn't quite like the use of `record_ip` here, since this is actually a hostname. I left it the way it is for discussion.

What would you think about changing this `record_ip` to `record_value`? Now would be the best time to do it if we were going to since 2.4 hasn't been released yet.